src/test/groovy/org/prebid/server/functional/util/privacy/GppConsent.groovy

```diff
@@ -0,0 +1,30 @@
+package org.prebid.server.functional.util.privacy
+
+import com.iab.gpp.encoder.GppModel
+import com.iab.gpp.encoder.section.TcfEuV2
+
+class GppConsent implements ConsentString {
+
+    private final GppModel gppModel
+
+    GppConsent() {
+        gppModel = new GppModel()
+    }
+
+    GppConsent setFieldValue(String sectionName = TcfEuV2.NAME,
+                             String fieldName = "ConsentLanguage",
```

Comment:
This default value looks too strange to be a default value.

Reply:
Also need clarification regarding this.